lib/vue-lib/src/design-system/general/JsonTreeExplorer/JsonTreeExplorer.vue

```diff
@@ -0,0 +1,146 @@
+<template>
```

Comment:
this is a general purpose JSON tree explorer - super useful for debugging stuff

Reply:
it also was the basis for the attributes panel since it has a similar recursive setup and shares some collapsing behaviour